src/fullnode.rs

```diff
@@ -223,13 +238,18 @@ impl Fullnode {
         let mut blob_recycler = BlobRecycler::default();
         blob_recycler.set_name("fullnode::Blob");
 
-        let rpu = Rpu::new(
+        let rpu = Some(Rpu::new(
             &bank,
-            node.sockets.requests,
-            node.sockets.respond,
+            node.sockets
```

Comment:
in other places, we're using Arcs for sockets...  we should pick one way or the other

in this PR, perhaps a TODO